pandas/core/frame.py

```diff
@@ -1428,7 +1428,10 @@ def to_latex(self, buf=None, columns=None, col_space=None, colSpace=None,
 
         if buf is None:
             return formatter.buf.getvalue()
-
+    
+    def _repr_latex_(self):
+        return self.to_latex()
```

Comment:
can you add a comment to note that this is for ipython? adding a reference to this GH issue would probably also be a good idea.